src/test/scala/de/unruh/isabelle/pure/Position.scala

```diff
@@ -0,0 +1,33 @@
+package de.unruh.isabelle.pure
+
+import de.unruh.isabelle.control.IsabelleTest.isabelle
+import org.scalatest.funsuite.AnyFunSuite
+
+
+class PositionTest extends AnyFunSuite {
```

Comment:
Can you add a test for `extract` and `extractRange`?

Reply:
Ok, added.